javagen/src/main/java/com/azure/autorest/Javagen.java

```diff
@@ -194,7 +196,12 @@ JavaPackage writeToTemplates(JavaSettings settings, CodeModel codeModel, Client
                             m.getProxyMethod().getExamples().forEach((name, example) -> {
                                 String filename = CodeNamer.toPascalCase(CodeNamer.removeInvalidCharacters(name));
                                 if (!protocolExampleNameSet.contains(filename)) {
-                                    ProtocolExample protocolExample = new ProtocolExample(m, c, client.getServiceClient(), builderName, filename, example);
+                                    // see code in ServiceClientBuilderTemplate
+                                    final String buildMethodName = (settings.shouldGenerateSyncAsyncClients() && !singleBuilder)
+                                            ? ("build" + c.getClassName())
+                                            : "buildClient";
```

Comment:
When builder got single client, template uses `buildClient` instead of full name of the client.